Cannot find module 'lolex'

Hello,

Since I moved my project from 1.1.0 to 1.2.0, I got the following error:

Error: Cannot find module 'lolex'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.require.resolve (internal/module.js:23:19)
    at framework (/…/node_modules/karma-sinon-chai/index.js:26:40)

--- a/package.json
+++ b/package.json
@@ -26,10 +26,11 @@
   ],
   "peerDependencies": {
     "chai": "^3.2.0",
-    "lolex": "^1.4.0",
     "sinon": "^1.17.2",
     "sinon-chai": "^2.8.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "lolex": "^1.5.0"
+  },
   "devDependencies": {}
 }

